src/linelast_solver.cpp

```diff
@@ -55,7 +55,11 @@ LinElastSolver::~LinElastSolver()
    delete globalMat;
    delete globalMat_hypre;
    delete mumps;
+   if (init_x != NULL)
```

Comment:
`delete init_x;` shouldn't have a problem as long as `init_x` is initialized as `NULL` in `include/linelast_solver.hpp`. Initialize it with `NULL` and remove the if clause.